### ISSUE TITLE
Very long blueprint page editor tabs prevent interacting with other tabs (css bug)

### ISSUE THREAD

**Issue by a1studmuffin** [getgrav/grav-plugin-admin#690]
I'm currently adding a gallery page. As part of that, I've added a page blueprint (which shows up as a "Gallery" tab) that contains forms for specifying a list of image categories and list of images (page media file, title, description, category) that should be in the gallery. It's all working great, except I've discovered when I add a lot of images (say 100 or so), that switching to another tab (eg. Content, Options, Advanced) results in those tabs not working as expected - they look fine but my mouse cursor is interacting with invisible elements of the Gallery tab despite it not being active. I can also see the scrollbar is very long in this case.

I've tracked the issue down to _tabs.scss:108:

`.tab-body {
    position:absolute;
    top:-9999px;
    opacity:0;
    width: 100%;
}`

If I change top to a more extreme number like -999999px then the issue goes away. I guess the tab content is greater than 9999px. :)

I imagine this issue should be easy to repro by creating a page blueprint containing a list of items, then filling it with a tonne of placeholder data.

Cheers!
Michael

**Comment by flaviocopes** [comment]
Mmmm, looks like the fix is easy but I cannot reproduce the original issue with Chrome on OSX - which browser are you trying?

**Comment by a1studmuffin** [comment]
I'm seeing the issue in all browsers I've tested with under Windows 10 x64 (latest browser updates):

Firefox 47.0.1
Chrome 51.0.2704.106 m
Edge 25.10586.0.0

Here's some screenshots highlighting the issue. Here's what it looks like normally (observe the new Gallery tab):

![screen shot 2016-07-12 at 13 09 54](https://cloud.githubusercontent.com/assets/6295625/16754021/04e394b2-4832-11e6-915f-18fae8a5f5dc.png)

In the next screenshot I've used the Inspector to select "tab-body-4" in the DOM (this is the container for the contents of the new "Gallery" tab I've added) and adjusted the CSS from:

`.tab-body {
    position:absolute;
    top:-9999px;
    opacity:0;
    width: 100%;
}`

to

`.tab-body {
    position:absolute;
    top:-9999px;
    opacity:1;
    width: 100%;
}`

![screen shot 2016-07-12 at 13 07 09](https://cloud.githubusercontent.com/assets/6295625/16753957/99373b24-4831-11e6-9cd4-42173da0f195.png)

So basically my cursor is interacting with the invisible elements of the Gallery tab which prevents me from using the other tabs correctly.

**Comment by rhukster** [comment]
Pretty sure we fixed this in Grav 1.1. Can you test. And confirm?

**Comment by a1studmuffin** [comment]
Excellent, yep upgraded to 1.1.0-rc3 and the issue is resolved. It appears instead of the `top: -9999px` hack the tab-body class uses `display: none` instead. Thanks all!

**Comment by flaviocopes** [comment]
Oh that's why I could not recreate it :) 👍
